### PR TITLE
Added pre-game plugins

### PR DESCRIPTION
**Pull request by SarehSadeghianasl**

### CODE REVIEW

**Review by adamsmj — CHANGES_REQUESTED** [review]
Hi Sareh, there's some problems with your request that you'll need to resolve before I can pull the changes into the repo:
- all of the files under frontend are generated and so can't be a part of the repo. Can you please remove them from your pull request and local git repo?
- likewise package.json and pom.xml shouldn't be included
- also, the XES lib should be loaded from maven, not from a lib folder, and so should also be removed from the pull request.

In other words, the only changed files that should be included in a push from you are those files under src/main

Can you please make those changes? I can then do the merge
